src/main/java/de/samply/share/broker/rest/Monitoring.java

```diff
@@ -224,18 +226,57 @@ public Response createQuery(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
           schema = @Schema(implementation = QueryObject.class))
           QueryObject queryObject) {
     try {
-      if (auth == null || !checkBasicAuth(auth)) {
+      if (auth == null || !checkBasicAuth(auth, "icinga.username", "icinga.password")) {
```

Comment:
Also here, I would call that monitoring.username and monitoring.password because the API we provide is for monitoring and Icinga is only one specific user of our common monitoring endpoint.